pallets/social/src/lib.rs

```diff
@@ -456,7 +453,7 @@ decl_module! {
       Self::add_blog_follower_and_insert_blog(owner.clone(), new_blog, true)?;
 
       <BlogIdsByOwner<T>>::mutate(owner.clone(), |ids| ids.push(blog_id));
-      BlogIdBySlug::insert(slug, blog_id);
+      BlogIdByHandle::insert(handle, blog_id);
```

Comment:
Maybe wrap `blog_id` into `Some(blog_id)`?
```suggestion
      BlogIdByHandle::insert(handle, Some(blog_id));
```

Reply:
The inner Option type is required in `::insert` by StorageMap